### PR TITLE
[RHOL-280] [1/2]Address compiler warnings for the Rholang project.

### PR DESCRIPTION
**Pull request by AbnerZheng**
As the ticket suggests, I divided into two PR. This is the first PR. 
@guardbotmk3 PTAL

### CODE REVIEW

**Comment by guardbotmk3** [comment]
@AbnerZheng For future reference, please use github's assign feature to assign me a review.

**Comment by AbnerZheng** [comment]
@MParlikar @guardbotmk3 Sorry, I have no privilege to assign reviewer.

File: rholang/src/main/scala/rholang/rosette/Roselang.scala [hunk 1]
```diff
@@ -337,7 +337,7 @@ trait RholangASTToTerm extends AllVisitor[VisitorTypes.R, VisitorTypes.A] {
       b match {
         case branch: Choice => {
           // bverity = 1, babsurdity = 0
-          val (bverity, babsurdity) =
+          val (_, babsurdity) =
```
[the remark this answers is not in the document]
Comment (AbnerZheng) [reply rholang/src/main/scala/rholang/rosette/Roselang.scala]: Done.

**Comment by birchmd** [comment]
@guardbotmk3 Please merge after confirming changes are good with you.

**Pull request merged**